tide_data_driven_component.install

```diff
@@ -37,3 +38,50 @@ function tide_data_driven_component_update_8001() {
     }
   }
 }
+
+/**
+ * Adding data_driven_component vocabulary.
+ */
+function tide_data_driven_component_update_8002() {
+  $configs = [
+    'taxonomy.vocabulary.data_driven_component' => 'taxonomy_vocabulary',
+    'field.field.taxonomy_term.data_driven_component.field_machine_name' => 'field_config',
+    'core.entity_form_display.taxonomy_term.data_driven_component.default' => 'entity_form_display',
+    'core.entity_view_display.taxonomy_term.data_driven_component.default' => 'entity_view_display',
+
+  ];
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_landing_page') . '/config/install'];
+  // Check if field already exported to config/sync.
+  foreach ($configs as $config => $type) {
+    $config_read = _tide_read_config($config, $config_location, TRUE);
+    $storage = \Drupal::entityTypeManager()->getStorage($type);
+    $config_entity = $storage->createFromStorageRecord($config_read);
+    $config_entity->save();
+  }
+}
+
+/**
+ * Adding data_driven_component terms.
+ */
+function tide_data_driven_component_update_8003() {
```

Comment:
These values are only valid for `content-vic`?

Reply:
Can this hook look at current `DATA_DRIVEN_OPTIONS` and create the terms based on that?

Reply:
! Thanks @anthony-malkoun , great point!

Reply:
Thanks @anthony-malkoun , Added https://github.com/dpc-sdp/tide_data_driven_component/pull/7/files#diff-bc568f94435aff13db5e6aa564e051f72987d50386320f959a3a79782cb9678bR90 !

Reply:
> These values are only valid for content-vic?

yes, should I put the hook into vicgovau_core module.

Reply:
I think it's fine here as if `DATA_DRIVEN_OPTIONS` is empty it just won't create any terms?

Reply:
yes. 
I just removed this one https://github.com/dpc-sdp/tide_data_driven_component/pull/7/commits/443a0b79623b03574291b5c46ada30328b4714e3#diff-bc568f94435aff13db5e6aa564e051f72987d50386320f959a3a79782cb9678bL68 , doesn't really need.